src/app/states/editor/page.mdx

```diff
@@ -0,0 +1,62 @@
+export const metadata = {
+  title: 'editor',
+  description: 'An app state of the editor. This section explains how to fetch and manipulate the `editor` state.',
+}
+
+# editor
+
+An [app state](/guides/flux-architecture) of the editor.
+You can obtain the `editor` state and integrate it with React components.{{ className: 'lead' }}
+
+
+---
+
+## Data Structure
+
+<Row>
+  <Col>
+    <Properties>
+      <Property name="changed" type="boolean | undefined">
+        `true` if the note has been changed. 
+        The editor will save the note if `changed` is true when it is closed.
+        Otherwise it will be`undefined` if it has not been changed yet since the app launched.
+      </Property>
+      <Property name="needsUpdatingDate" type="boolean | undefined">
+        Whether the note's `updatedAt` field needs to be updated or not.
+      </Property>
+      <Property name="viewMode" type="preview | sideBySide | edit">
+        The editor view mode which represents either `'preview'`, `'sideBySide'` or `'edit'`.
+      </Property>
+      <Property name="openingNoteId" type="string | undefined">
+        When the app is trying to open a note, it indicates its' note ID, otherwise `undefined`.
+      </Property>
+      <Property name="readOnly" type="boolean | undefined">
+        true if the editor is in read only mode.
+      </Property>
+      <Property name="savingNoteId" type="string | undefined">
+        When the app is saving the note, it indicates its' node ID, otherwise will be `undifined`.
```

Comment:
```suggestion
        When the app is saving the note, it indicates its' note ID, otherwise will be `undefined`.
```
A typo in the old documentation